databricks/resource_databricks_workspace_conf.go

```diff
@@ -0,0 +1,198 @@
+			"enable_ip_access_lists": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			// "sidebar_logo_active": {
+			// 	Type:     schema.TypeBool,
+			// 	Optional: true,
+			// 	Default:  false,
+			// },
+			// "home_page_welcome_message": {
+			// 	Type:     schema.TypeString,
+			// 	Optional: true,
+			// 	Default:  "",
+			// },
+			// "sidebar_logo_text": {
+			// 	Type:     schema.TypeString,
+			// 	Optional: true,
+			// 	Default:  "",
+			// },
+			// "sidebar_logo_inactive": {
+			// 	Type:     schema.TypeBool,
+			// 	Optional: true,
+			// 	Default:  false,
+			// },
+			// "home_page_logo": {
+			// 	Type:         schema.TypeString,
+			// 	Optional:     true,
+			// 	Default:      "",
+			// 	ValidateFunc: validation.IsURLWithHTTPorHTTPS,
+			// },
+			// "home_page_logo_width": {
+			// 	Type:     schema.TypeString,
+			// 	Optional: true,
+			// 	Default:  "",
+			// },
+			// "product_name": {
+			// 	Type:     schema.TypeString,
+			// 	Optional: true,
+			// 	Default:  "",
+			// },
+			// "login_logo": {
+			// 	Type:         schema.TypeString,
+			// 	Optional:     true,
+			// 	Default:      "",
+			// 	ValidateFunc: validation.IsURLWithHTTPorHTTPS,
+			// },
+			// "login_logo_width": {
+			// 	Type:     schema.TypeString,
+			// 	Optional: true,
+			// 	Default:  "",
+			// },
+			// "custom_references": {
+			// 	Type:     schema.TypeString,
+			// 	Optional: true,
+			// 	Default:  "",
+			// },
+		},
+	}
+}
+
+func resourceWorkspaceConfCreateOrUpdate(d *schema.ResourceData, m interface{}) error {
+	client := m.(*service.DBApiClient)
+
+	wsConfMapBuilder := model.WorkspaceConfRequestMapBuilder{}
+	wsConfMap := wsConfMapBuilder.Build(
+		// model.WithProductName(d.Get("product_name").(string)),
+		// model.WithSidebarLogoText(d.Get("sidebar_logo_text").(string)),
+		// model.WithLoginLogo(d.Get("login_logo").(string)),
+		// model.WithLoginLogoWidth(d.Get("login_logo_width").(string)),
+		// model.WithHomePageWelcomeMessage(d.Get("home_page_welcome_message").(string)),
+		// model.WithHomePageLogo(d.Get("home_page_logo").(string)),
+		// model.WithHomePageLogoWidth(d.Get("home_page_logo_width").(string)),
+		// model.WithSidebarLogoActive(strconv.FormatBool(d.Get("sidebar_logo_active").(bool))),
+		// model.WithSidebarLogoInactive(strconv.FormatBool(d.Get("sidebar_logo_inactive").(bool))),
+		// model.WithCustomReferences(d.Get("custom_references").(string)),
+		model.WithEnableIpAccessLists(strconv.FormatBool(d.Get("enable_ip_access_lists").(bool))),
+	)
+	err := client.WorkspaceConfigurations().Update(wsConfMap)
+	if err != nil {
+		return err
+	}
+
+	d.SetId("workspace_configs")
+
+	return resourceWorkspaceConfRead(d, m)
+}
+
+func resourceWorkspaceConfRead(d *schema.ResourceData, m interface{}) error {
+	client := m.(*service.DBApiClient)
+	resp, err := client.WorkspaceConfigurations().Read(model.AllWorkspaceConfKeys)
+	if err != nil {
+		return err
+	}
```

Comment:
Can we verify that this api does not have 404s and add that as a comment. This can identify that there is no configuration drift.

Reply:
Adding comment, only 200 or 400 for Get and 200, 400, 401, and 500 on patch